### PR TITLE
Allow empty constant query values. Fixes #391.

### LINKED ISSUE #391: QB validator does not allow for a constant value of empty string
Fun workaround though: instead of using "is equal to" with a blank constant, use "is in" with a constant of ['']

### PR DESCRIPTION
**Pull request by jrbotros**
Sets the default CPF value to an empty string and removes input validation on constant values.

### CODE REVIEW

File: jia/jia/static/app/board/querybuilder.js [hunk 1]
```diff
@@ -402,7 +402,7 @@ function (findObjectInListBasedOnKey, makeComplaint, revokeComplaint,
       _.each(scope.cpf.args, function (arg, index) {
         var type = scope.cpf.func.options[index].type;
         var cpf = {
-          'cpf_type': type
+          'cpf_type': type,
```
Comment (derek-schultz) [inline jia/jia/static/app/board/querybuilder.js]: No comma

File: jia/jia/static/app/board/querybuilder.js [hunk 2]
```diff
@@ -608,6 +598,7 @@ function (findObjectInListBasedOnKey, makeComplaint, revokeComplaint,
       {name: 'Function', type: 'function'}
     ];
     $scope.cpf.type = $scope.types[0];
+    $scope.cpf.value = '';
```
Comment (derek-schultz) [inline jia/jia/static/app/board/querybuilder.js]: Good catch

File: jia/jia/static/app/board/querybuilder.js (hunk 1 above)
Comment (jrbotros) [inline jia/jia/static/app/board/querybuilder.js]: :white_check_mark:

**Comment by derek-schultz** [comment]
Looks good! Squash and I'll merge it.

**Pull request merged**